sqllineage/core.py

```diff
@@ -91,113 +91,116 @@ def analyze(self, stmt: Statement) -> LineageResult:
             pass
         else:
             # DML parsing logic also applies to CREATE DDL
-            self._extract_from_DML(stmt)
+            self._extract_from_dml(stmt)
         return self._lineage_result
 
-    def _extract_from_DDL_DROP(self, stmt: Statement) -> None:
+    def _extract_from_ddl_drop(self, stmt: Statement) -> None:
         for table in {
             Table.create(t) for t in stmt.tokens if isinstance(t, Identifier)
         }:
             self._lineage_result.drop.add(table)
 
-    def _extract_from_DDL_ALTER(self, stmt: Statement) -> None:
+    def _extract_from_ddl_alter(self, stmt: Statement) -> None:
         tables = [Table.create(t) for t in stmt.tokens if isinstance(t, Identifier)]
         keywords = [t for t in stmt.tokens if t.ttype is Keyword]
         if any(k.normalized == "RENAME" for k in keywords) and len(tables) == 2:
             self._lineage_result.rename.add((tables[0], tables[1]))
 
-    def _extract_from_DML(self, token: Token) -> None:
-        source_table_token_flag = (
-            target_table_token_flag
-        ) = temp_table_token_flag = False
+    def _extract_from_dml(self, token: Token) -> None:
+        source_table_token_flag = False
+        target_table_token_flag = False
+        temp_table_token_flag = False
         for sub_token in token.tokens:
+            if self.__token_negligible_before_tablename(sub_token):
+                continue
+
             if isinstance(sub_token, TokenList):
-                self._extract_from_DML(sub_token)
+                self._extract_from_dml(sub_token)
+
             if sub_token.ttype in Keyword:
-                if any(
-                    re.match(regex, sub_token.normalized)
-                    for regex in SOURCE_TABLE_TOKENS
```

Comment:
This part is following [black ](https://github.com/psf/black) code style. I'd like to stay with that.

Reply:
Understood.  I've pushed a commit with black output.